docker/templates/killbill/light/killbill.properties.template

```diff
@@ -0,0 +1,14 @@
+org.killbill.security.shiroNbHashIterations=1
+org.killbill.jaxrs.threads.pool.nb=5
+org.killbill.payment.plugin.threads.nb=1
```

Comment:
You are going from one extreme to the next :-)

Reply:
Well, for serverless deployments, the payment integration doesn't really work out of the box:

* Ruby plugins are too memory hungry
* OSS Java payment plugins are for gateways which require IP source whitelisting (unpredictable)

It's really a subscription-only profile which should be deployed :-)